Src/Models/CBR.cpp

```diff
@@ -2,23 +2,30 @@
 
 #include <set>
 #include <fstream>
+#include <sys/stat.h>
```

Comment:
Do not include POSIX headers without platform-specific include guards. If code is meant to be compiled on all platforms, it needs to depend on only the libraries we use and the C++17 standard.

Reply:
I did not know this.
Good to know.
Turns out, this include is actually a leftover of testing and isn't actually used.
It has been removed now.